Makefile

```diff
@@ -20,6 +20,10 @@ else
 	VERSION = $(TAG)
 endif
 
+ifeq ($(RP_IMAGE_ACR),)
```

Comment:
should this have an else that sets REGISTRY to the redhat value or to the value of RP_IMAGE_ACR? I'm not clear on how this is meant to be used.